tests/e2e/specs/substance.js

```diff
@@ -697,6 +703,81 @@ describe("The substance page authenticated access", () => {
       .its("request.body.data.relationships.queryStructureType.data.id")
       .should("contain", "markush-query");
   });
+  it("should reset changes to the Ketcher window", () => {
```

Comment:
This test seems great and passes.  Do you think another one that tests functionality on Marvin would be good?

Reply:
We had talked a little bit about using "Export" on the Marvin window to check what compound is drawn (or not drawn) and you'd mentioned it was pretty sketchy for Cypress to access the export window contents. 

The Marvin test (ln 754) currently adds Cyclohexane to the window then resets. The 'pass' condition is that the "Save Compound" & "Reset Compound" buttons are disabled after reset. I'm not sure what else I could do to check the contents of the window aside from the "Export" function.

Reply:
You know I completely missed the following test `it("should reset changes to the Marvin window", () => { })`.  That test was exactly what I am looking for.  

Sorry for the confusion.